tasks/ModelTask.cpp

```diff
@@ -89,13 +89,16 @@ void ModelTask::setupLinks()
         if (it->target_link != _world_frame.get())
             exported_link.target_link_ptr = model->GetLink( it->target_link );
         exported_link.port_name = it->port_name;
+        exported_link.port_period = it->port_period;
 
         if (exported_link.source_link != _world_frame.get() && !exported_link.source_link_ptr)
         { gzthrow("ModelTask: cannot find exported source link " << it->source_link << " in model"); }
         else if (exported_link.target_link != _world_frame.get() && !exported_link.target_link_ptr)
         { gzthrow("ModelTask: cannot find exported target link " << it->target_link << " in model"); }
         else if (it->port_name.empty())
         { gzthrow("ModelTask: no port name given in link export"); }
+        else if (it->port_period.isNull())
```

Comment:
I think you should allow this, using the current "update at every simulation cycle" meaning.